app/assets/templates/layout/navbar_right.html.slim

```diff
@@ -1,12 +1,12 @@
 ul.nav.navbar-nav.navbar-right
   li.my-account
-    a(ng-click="goToMyAccount()") My Account
+    a(ng-click="goToMyAccount()")= I18n.t 'sprangular.my_account'
```

Comment:
Good point, it will be faster to use rails `I18n.t`.

Though, wondering if this will work with the asset pipeline.

Like if I do `rake assets:precompile`, it will generate only one version of each template using the `current_locale` of `rake`. So it might not work in production